src/fs/dir.rs

```diff
@@ -111,12 +111,12 @@ impl Dir {
     #[cfg(windows)]
     fn _open_dir(&self, path: &Path) -> io::Result<Self> {
         use std::os::windows::fs::OpenOptionsExt;
-        use winapi::um::winbase::FILE_FLAG_BACKUP_SEMANTICS;
+        use winx::file::Flags;
         self.open_with(
             path,
             OpenOptions::new()
                 .read(true)
-                .attributes(FILE_FLAG_BACKUP_SEMANTICS),
+                .attributes(Flags::FILE_FLAG_BACKUP_SEMANTICS.bits()),
```

Comment:
Can this use `dir_options()` instead of a manual `OpenOptions`? It seems like that might also eliminate the need for `cap-std` and `cap-async-std` to depend directly on `winx`.

Reply:
Excellent idea, thanks!